### ISSUE TITLE
Training interrupted after first epoch

### ISSUE THREAD

**Issue by skanderZitoun** [XifengGuo/CapsNet-Keras#127]
During training on custom dataset (80, 80, 3) everything looks fine , once the first epoch is complete this error occured :

```
WARNING:tensorflow:`batch_size` is no longer needed in the `TensorBoard` Callback and will be ignored in TensorFlow 2.0.
Epoch 1/30
62/62 [==============================] - ETA: 0s - loss: 0.8135 - capsnet_loss: 0.8096 - decoder_loss: 0.0393 - capsnet_accuracy: 0.2384 

InvalidArgumentError:  Specified a list with shape [3,32768,8,1] from a tensor with shape [3,22528,8,1]      [[node functional_13/digitcaps/map/TensorArrayUnstack/TensorListFromTensor (defined at <ipython-input-5-ed04f28927aa>:48) ]] [Op:__inference_test_function_5199]

Errors may have originated from an input operation. Input Source operations connected to node functional_13/digitcaps/map/TensorArrayUnstack/TensorListFromTensor:  functional_13/digitcaps/Tile (defined at <ipython-input-5-ed04f28927aa>:40)

Function call stack: test_function
```
Can anyone help me fix it please ?

**Comment by davidbacelj** [comment]
Hi, did you manage to solve this issue?
